vagrant/guest/bin/m2setup.sh

```diff
@@ -76,6 +76,13 @@ for arg in "$@"; do
                 exit -1
             fi
             ;;
+        --proj-version=*)
+            PROJ_VERSION="${arg#*=}"
+            if [[ ! "$PROJ_VERSION" =~ ^[a-z0-9\.-]+$ ]]; then
```

Comment:
I questioned whether such a strict validation was necessary. What do you think about just changing the regex to `^.+$`